launch/usb_cam-test.launch

```diff
@@ -1,15 +0,0 @@
-<launch>
```

Comment:
we don't need to remove this file.
we should keep the backward compatibility, if possible.

Reply:
yes.
or you can keep the old file as symbolic link

Reply:
Excellent, so we will keep `test_img_view.launch` with new contents